content/en/blog/news/2022-gsoc-work-report.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "GSoC 2022 Final Report: Building Jenkins X UI"
+date: 2022-11-13
+draft: false
+description: This project replaces the older UI with the new UI with more powerful features.
+categories: [blog]
+keywords: [Community, jenkins-x, gsoc, 2022]
+slug: "GSoC-2022-work-report"
+aliases: []
+author: Rajat Gupta
+---
+
+## Jenkins X New UI
+It is a helm chart that you can install with the other tools of jx. It is build using [Golang](https://go.dev/) as backend and [Sveltekit](https://kit.svelte.dev/) as frontend, both backend and frontend are build together and are used in the same container.
```

Comment:
Start saying: "It is a web application built with Golang ... (next sentence). Add info on helm chart next. ("It is available as a helm chart so you can install like any other Kubernetes component") ("But needs Jenkins X CRDs to work properly?!)